.gitignore

```diff
@@ -1 +1,3 @@
-*/build
\ No newline at end of file
+*/build
+
+.idea/
```

Comment:
Everything looks good, but one final question, what's the reason for this?

Reply:
I use Phpstorm thats why the ignore .idea and */build because i don’t want to commit the built version.

Reply:
I gotcha, I wasn't aware, never used that IDE. Thanks! :)